drivers/dma/dma_mcux_lpc.c

```diff
@@ -461,6 +478,11 @@ static int dma_mcux_lpc_configure(const struct device *dev, uint32_t channel,
 
 	k_spin_unlock(&configuring_otrigs, otrigs_key);
 
+	if (config->complete_callback_en) {
+		complete_callback = true;
+	}
```

Comment:
just curious is it possible to do `complete_callback = (bool) config->complete_callback_en` or something like that?